### DISCUSSION TITLE
Design Questions - Icon in Tables, Selection Row, Wasted Space

### DISCUSSION THREAD

**Discussion by TheSpectre568** [Budibase/budibase#14003] [Help] [answered]
Hey together!

I'm started to try out BudiBase to create a tool for a club.
For now it's working fine, but i have a few questions with the design of tables.

My goal is to have a few tables that look as clean and nice, as in some 1-year-old Videos from BudiBase itself.

Like that:
![image](https://github.com/Budibase/budibase/assets/149518757/75162a30-321c-4382-989f-e1436988be3e)

But my tables look a lot more different.
And the main points are:
1. - Can i remove the icons on the Heading of the table?
2. - Can i remove the selection coloumn on the left of the table?
3. - Can i make to table somehow a bit more dynamic, as that the row adapt on the width of the table? And that the the tables height is not more than the space it really needs..?

I marked all three points here:
![image](https://github.com/Budibase/budibase/assets/149518757/4382d27a-4824-49d5-91bf-bfb169fa27c6)

If this would somehow possible it would be awesome!

Best Regards

**Comment by mikesealey** [comment, marked as answer]
Hey @TheSpectre568 

Over time the appearance of Budibase has changed, but ultimately the goal is for the appearance of _your app_ to be in _your hands_. 

There are a couple of ways you could approach this is using a Custom Plugin - one community favourite is the SuperTable plugin by @poirazis. You can see a (non-exhaustive) list of plugins here: https://github.com/Budibase/plugins and you can read our documentation on how to get started with custom plugins here: https://docs.budibase.com/docs/custom-plugin
 
The alternative option is using an `<embed>` to do some custom styling. https://docs.budibase.com/docs/embed where you could write some custom styling to target specific elements in your app. You'll want to make good use of your browser's dev-tools in order to target the right components. Make sure to also append each rule with `!important` in order to override the existing styling rules. You might be best starting off with `border: 3px solid red !important;` when targeting an element - that way you can be sure which element you're working on. From there you could set the visibility of the icons to none, for example.
